### ISSUE TITLE
beam search log prob

### ISSUE THREAD

**Issue by seanie12** [atulkum/pointer_summarizer#26]
I think final_dist from decoder is not log scale value, because it comes from the softmax. So before instantiate new Beam in beam_search() at decode.py, log(e.g torch.log ) needs to be applied.

**Comment by atulkum** [comment]
You are right. I fixed it here https://github.com/atulkum/pointer_summarizer/blob/master/training_ptr_gen/decode.py#L161

Did you see any performance improvement after this change?

**Comment by seanie12** [comment]
sorry, i didn't run the code
